src/MathProgBase.jl

```diff
@@ -4,6 +4,7 @@ module MathProgBase
 
 include(joinpath(dirname(@__FILE__),"SolverInterface","SolverInterface.jl"))
 using .SolverInterface
+using Compat
```

Comment:
Among all the tags in `Compat` there is no commit which includes `issymmteric`, @tkelman how do I find the version here?